raygun.gemspec

```
@@ -23,4 +23,5 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "bundler", "~> 2.0"
   gem.add_development_dependency "rake", "~> 13.0"
+  gem.add_development_dependency "rubocop", "~> 0.79"
```

Since we are not committing a `Gemfile.lock` I think we should pin rubocop here. New versions of rubocop often contain breaking changes, unfortunately.

```suggestion
  gem.add_development_dependency "rubocop", "0.79.0"
```